config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "users#posts", as: :user_posts
+  get "/users/:id/posts/:id", to: "users#post", as: :user_post
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you add the routes using the `resources` method which is easier to manage and more easily readable and understandable 👍 You can see the example below ⬇️

```RB
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```